linkerd/core/src/main/scala/io/buoyant/linkerd/Router.scala

```diff
@@ -123,19 +132,42 @@ trait RouterConfig {
   def interpreter: InterpreterConfig =
     _interpreter.getOrElse(defaultInterpreter)
 
+  /*
+   * bindingTimeoutMs limits name resolution.
+   */
+
   @JsonProperty("bindingTimeoutMs")
   var _bindingTimeoutMs: Option[Int] = None
 
   @JsonIgnore
   def bindingTimeout = _bindingTimeoutMs.map(_.millis).getOrElse(10.seconds)
 
+  /*
+   * responseClassifier categorizes responses to determine whether
+   * they are failures and if they are retryable.
+   */
+
+  @JsonProperty("responseClassifier")
+  var _responseClassifier: Option[ResponseClassifierConfig] = None
+
+  @JsonIgnore
+  def baseResponseClassifier: ResponseClassifier =
+    ResponseClassifier.Default
+
+  @JsonIgnore
+  def responseClassifier: ResponseClassifier =
+    _responseClassifier.foldLeft(baseResponseClassifier) {
+      case (base, config) => config.mk orElse base
```

Comment:
neither of these are functionally equivalent ;) but I can change this since it seems to be confusing.